### ISSUE TITLE
Annotations do not show in Editor and Viewer, daily build PDF4QT_LaunchPad-1e79871-x86_64.AppImage

### ISSUE THREAD

**Issue by SnowballV** [JakubMelka/PDF4QT#191]
OS is Kubuntu 22.04.4 LTS.

I tested daily build PDF4QT_LaunchPad-1e79871-x86_64.AppImage.
https://github.com/JakubMelka/PDF4QT/actions/runs/9681243851
I tested the Editor and the Viewer.

In two different PDF documents, I made a few annotations for testing: I inserted rectangles, highlighted text, underlined text, and added a sticky note.
Annotations do show in the Thumbnails bar, but not in the main window or field -- not in the Editor, and when the document is saved the annotations don't show in the Viewer either.
The annotations do show when I open the saved documents in Okular, or in Evince.

Settings for rendering options are correct: View\ Rendering Options\ Antialiasing, Text Antialiasing, and Display Annotations are enabled.

Do you have suggestions for debugging this issue, annotations not showing in Editor and Viewer?

**Comment by JakubMelka** [comment]
@SnowballV, could you please try a newest build https://github.com/JakubMelka/PDF4QT/actions/runs/9715015764 ? I have fixed a bug in annotation display probably yesterday.

**Comment by SnowballV** [comment]
@JakubMelka,
Thanks.
I see pdf4qt-Windows.zip and ubuntu-deb-package are offered, but not an AppImage.
I would like to test the AppImage, not install the deb.

**Comment by JakubMelka** [comment]
@SnowballV, I have prepared a new install here, please try it:
https://github.com/JakubMelka/PDF4QT/actions/runs/9723831736

**Comment by SnowballV** [comment]
@JakubMelka,
Thank you very much.

I tested the new PDF4QT_LaunchPad-b6fda40-x86_64.AppImage.
https://github.com/JakubMelka/PDF4QT/actions/runs/9723831736
I tested the Editor and the Viewer.

In two different PDF documents, I created new annotations for testing: I highlighted text, underlined text, added a sticky note, inserted rectangles, and inserted inline text.

Annotations now show correctly in the Editor and in the Viewer.
Annotations show correctly in Okular and Evince as well.
Annotations created with the previous PDF4QT build, or made with PDF-XChange Editor 10 years ago, all show correctly also.

As the reported problem in PDF4QT_LaunchPad-1e79871-x86_64.AppImage is resolved in PDF4QT_LaunchPad-b6fda40-x86_64.AppImage, I will close this issue.

Thanks very much, once more.
